lib/wcc/contentful/store.rb

```diff
@@ -30,9 +30,9 @@
 module WCC::Contentful::Store
   SYNC_STORES = {
     memory: ->(_config) { WCC::Contentful::Store::MemoryStore.new },
-    postgres: ->(_config) {
+    postgres: ->(_config, *_options) {
```

Comment:
since `connection_options` is a hash, should this be `**_options`?

Reply:
since these are now used - should no longer have `_` prefix

also, want to write a spec for the postgres store ensuring both array & hash config options work?